xc7/tests/litex/minilitex/README.md

```diff
@@ -0,0 +1,5 @@
+# MiniLitex - Litex BaseSoC
+
+This test features a Mini Litex design with a VexRiscv CPU in the minimal variant.
+The firmware is compiled into the bitstream and the ROM and SRAM memories are instantiated and initialized on the FPGA (no DDR RAM controller).
+
```

Comment:
Please add instructions on how to recreated the verilog and similar files.

Reply:
Done